### ISSUE TITLE
Players crashing with this mod installed

### ISSUE THREAD

**Issue by BackgroundPony** [MineLittlePony/MineLittlePony#168]
Only players with this mod are crashing on my server, and they crash at night time. here are the crash logs from two of the players
[crash-2020-05-23_23.02.51-client.txt](https://github.com/MineLittlePony/MineLittlePony/files/4770898/crash-2020-05-23_23.02.51-client.txt)
[crash-2020-06-07_07.58.34-client.txt](https://github.com/MineLittlePony/MineLittlePony/files/4770903/crash-2020-06-07_07.58.34-client.txt)

**Comment by Andy-K-Sparklight** [comment]
Should also install this MOD on your server, I guess?

(I am not a member of the project, but I have a sincere heart)

**Comment by Sollace** [comment]
No. Do not install Mine Little Pony on the server.

**Comment by Andy-K-Sparklight** [comment]
Sorry for my mistake, last time I installed it on server then nobody crashed, but I just found out that it’s because of my server settings.

~Looks like something wrong with the **Random** when rendering , is there a negative num in the bound? (I guessed by the crash report)~@Sollace

**Comment by backgroundman** [comment]
Forgot to mention the crashes are much more frequent and happen every day. These are just two of the logs I was sent. 
Andy, you mention nobody crashed because of your server's settings. Could you clarify?

**Comment by Sollace** [comment]
This is arecurring issue, mostly with how Mojang has their arrow renderer set up.

tl;dr They don't like model parts with no cuboids. I've tried fixing it multiple times in Mson, but evidently it's still an issue.

**Comment by Andy-K-Sparklight** [comment]
> Forgot to mention the crashes are much more frequent and happen every day. These are just two of the logs I was sent.
> Andy, you mention nobody crashed because of your server's settings. Could you clarify?

Eeyup...My MC version is 1.14. I mean, I wrote some extra pre-checks to make sure nopony cheats (Including mods) when playing with my server, I allowed some mods, including MineLP, but I type the name wrong (**my**littlepony instead of minelittlepony). After I fix it, the players can load MineLP.

**Comment by Andy-K-Sparklight** [comment]
Not because of the settings
